New q audio

AndreyTepaykin opened the pull request.

EGreg commented on platform/classes/Q/File.php:
```diff
@@ -59,6 +59,9 @@ static function save($params)
 		$tailUrl = $subpath ? "$path/$subpath/$name" : "$path/$name";
 
+		// need to send to Q/file/save after event
+		$audio = (bool)$params['audio'];
```
Look, `$params['audio']` may be undefined! Please fix it. !empty($params['audio']).

AndreyTepaykin commented:
Fixed.
Please review again.

2017-12-21 3:13 GMT+03:00 Gregory Magarshak <notifications@github.com>:

> *@EGreg* requested changes on this pull request.
>
> I always same thing, to program defensively!
> <https://en.wikipedia.org/wiki/Defensive_programming> When writing code,
> think: "what if this variable is undefined" or "what if this precondition
> is not met"?
>
> Another thing: why did you change the images and removed the round
> outlines? I was fine with the round outlines. Can you return them if you
> didn't have a good reason?
> ------------------------------
>
> In platform/classes/Q/File.php
> <https://github.com/Qbix/Platform/pull/647#discussion_r158168641>:
>
> > @@ -59,6 +59,9 @@ static function save($params)
>
>  		$tailUrl = $subpath ? "$path/$subpath/$name" : "$path/$name";
>
> +		// need to send to Q/file/save after event
> +		$audio = (bool)$params['audio'];
>
> Look, $params['audio'] may be undefined! Please fix it.
> !empty($params['audio']).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Qbix/Platform/pull/647#pullrequestreview-84935702>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGlt4YEfTewQ-SmG4hkQMNvP3Kdxvdc4ks5tCaKcgaJpZM4RFs5B>
> .
>

EGreg commented:
I need one more thing: please return the graphics that have circles, or explain why did you remove the circles?
